eng/pipelines/templates/stages/platform-matrix-live.json

```diff
@@ -0,0 +1,133 @@
+{
+    "displayNames": {
+        "_": ""
+    },
+    "include": [
+        {
+            "StaticConfigs": {
+                "MacOS_x64_with_unit_test": {
+                    "Pool": "Azure Pipelines",
+                    "OSVmImage": "macOS-10.15",
+                    "VCPKG_DEFAULT_TRIPLET": "x64-osx",
+                    "CmakeArgs": " -DBUILD_TESTING=ON -DRUN_LONG_UNIT_TESTS=ON -DBUILD_PERFORMANCE_TESTS=ON -DBUILD_SAMPLES=ON",
+                    "AZURE_CORE_ENABLE_JSON_TESTS": 1,
+                    "BuildArgs": "-j 4"
+                }
+            }
+        },
+        {
+            "StaticConfigs": {
+                "_": {
+                    "VCPKG_DEFAULT_TRIPLET": "x64-linux",
+                    "BuildArgs": "-j 4",
+                    "Pool": "azsdk-pool-mms-ubuntu-1804-general",
+                    "OSVmImage": "MMSUbuntu18.04"
+                }
+            },
+            "BuildConfig": {
+                "Linux_x64_gcc5_with_unit_test": {
+                    "CmakeEnvArg": "CC=/usr/bin/gcc-5 CXX=/usr/bin/g++-5 cmake",
+                    "CmakeArgs": " -DBUILD_TESTING=ON -DRUN_LONG_UNIT_TESTS=ON"
+                },
+                "Linux_x64_with_unit_test": {
+                    "CmakeArgs": " -DBUILD_TESTING=ON -DBUILD_PERFORMANCE_TESTS=ON -DRUN_LONG_UNIT_TESTS=ON -DCMAKE_BUILD_TYPE=Debug -DBUILD_CODE_COVERAGE=ON",
+                    "AptDependencies": "gcovr lcov",
+                    "CODE_COVERAGE": "enabled",
+                    "CODE_COVERAGE_COLLECT_ONLY": 1,
+                    "AZURE_CORE_ENABLE_JSON_TESTS": 1
+                }
+            }
+        },
+        {
+            "StaticConfigs": {
+                "_": {
+                    "BuildArgs": "-j 4",
+                    "Pool": "azsdk-pool-mms-ubuntu-1804-general",
+                    "OSVmImage": "MMSUbuntu18.04",
```

Comment:
Ubuntu 20 is, in all cases of the live tests, specified as Ubuntu 18 in reality: https://github.com/Azure/azure-sdk-for-cpp/pull/3553/files#diff-aaa8594d515b0817d42705241d82c9f1a7aa598afefe8fb59e53be5d5b6b3994L55